src/compilation/compiler.rs

```diff
@@ -316,10 +368,15 @@ impl Compiler {
         }
     }
 
+    /// Compiles an expression being used as a statement (by ignoring its value).
     unsafe fn compile_expr_statement(&mut self, expr: &Expr) {
+        // TODO: Bug: the expected type shouldn't be Void; rather, it should be Any?
+        //  Well, unless we want people to write _ = foo() instead of just foo(). But
+        //  then we should implement _.
```

Comment:
Oh good catch. Another easy thing for us to fix together sometime soon!

Reply:
:D